EnglishFilter/sections/content_blocker.txt

```diff
@@ -1337,6 +1337,10 @@ gcertificationcourse.com###sticky-ad
 @@||contextual.media.net/dmedianet.js$domain=politics-dz.com
 @@||politics-dz.com/wp-content/plugins/ad-inserter/*/*.js
 @@||politics-dz.com/wp-content/plugins/ad-inserter/images/ads.png
+! https://github.com/AdguardTeam/AdguardFilters/issues/74617
```

Comment:
Antiadblock still triggers with these rules in user rules

Reply:
@Sergey-Lyapin Suggest me new rules for this. I have no idea right now.

Reply:
Fixed.